src/archiefbeheercomponent/conf/locale/nl/LC_MESSAGES/django.po

```diff
@@ -325,148 +379,168 @@ msgstr ""
 msgid "User, who performed review"
 msgstr "Gebruiker die de review uitvoerde"
 
-#: archiefvernietigingscomponent/destruction/models.py:259
-#: archiefvernietigingscomponent/destruction/models.py:286
-#: archiefvernietigingscomponent/destruction/models.py:315
+#: archiefbeheercomponent/destruction/models.py:281
+#: archiefbeheercomponent/destruction/models.py:308
+#: archiefbeheercomponent/destruction/models.py:337
 msgid "text"
 msgstr "tekst"
 
-#: archiefvernietigingscomponent/destruction/models.py:262
+#: archiefbeheercomponent/destruction/models.py:284
 msgid "Review comments on the destruction list in general"
 msgstr "Algemene opmerkingen op de vernietigingslijst"
 
-#: archiefvernietigingscomponent/destruction/models.py:271
-#: archiefvernietigingscomponent/destruction/models.py:283
-#: archiefvernietigingscomponent/destruction/models.py:306
+#: archiefbeheercomponent/destruction/models.py:293
+#: archiefbeheercomponent/destruction/models.py:305
+#: archiefbeheercomponent/destruction/models.py:328
 msgid "destruction list review"
 msgstr "vernietigingslijstreview"
 
-#: archiefvernietigingscomponent/destruction/models.py:272
+#: archiefbeheercomponent/destruction/models.py:294
 msgid "destruction list reviews"
 msgstr "vernietigingslijstreviews"
 
-#: archiefvernietigingscomponent/destruction/models.py:289
+#: archiefbeheercomponent/destruction/models.py:311
 msgid "Text of the comment to the review"
 msgstr "Text van de opmerking aan de review"
 
-#: archiefvernietigingscomponent/destruction/models.py:294
+#: archiefbeheercomponent/destruction/models.py:316
 msgid "destruction list review comment"
 msgstr "vernietigingslijstreview opmerking"
 
-#: archiefvernietigingscomponent/destruction/models.py:295
+#: archiefbeheercomponent/destruction/models.py:317
 msgid "destruction list review comments"
 msgstr "vernietigingslijstreview opmerkingen"
 
-#: archiefvernietigingscomponent/destruction/models.py:318
+#: archiefbeheercomponent/destruction/models.py:340
 msgid "Review comments on the destruction list item"
 msgstr "Opmerkingen op de item in de vernietigingslijst"
 
-#: archiefvernietigingscomponent/destruction/models.py:321
+#: archiefbeheercomponent/destruction/models.py:343
 msgid "suggestion"
 msgstr "suggestie"
 
-#: archiefvernietigingscomponent/destruction/models.py:325
+#: archiefbeheercomponent/destruction/models.py:347
 msgid "destruction list item review"
 msgstr "vernietingslijst item review"
 
-#: archiefvernietigingscomponent/destruction/models.py:326
+#: archiefbeheercomponent/destruction/models.py:348
 msgid "destruction list item reviews"
 msgstr "vernietingslijst item reviews"
 
-#: archiefvernietigingscomponent/destruction/models.py:343
+#: archiefbeheercomponent/destruction/models.py:365
 msgid "order"
 msgstr "volgorde"
 
-#: archiefvernietigingscomponent/destruction/models.py:346
+#: archiefbeheercomponent/destruction/models.py:366
+msgid "assigned on"
+msgstr "toegewezen op"
```

Comment:
```suggestion
msgstr "toegewezen aan"
```

Althought the translation is probably correct, the English sounds weird as well? "Assigned on" should probably be "Assigned to", and the Dutch: "Toegewezen aan"

Reply:
@joeribekker This is a Datetime field, so 'Assigned on' is specifying _when_ someone was assigned (rather than _who_ was assigned)

Reply:
Thanks, makes sense.